README.md

```diff
@@ -217,6 +218,20 @@ Generally, if you want to release a prerelease of a repository, and it is an NPM
 
 Please note that in case of a prerelease the `sync-semver-tags` input will be treated as `false`, even if it's set to `true`. This because we don't want to update the main version tags to the latest prerelease commit but only to the latest official release.
 
+## Provenance
+
+NPM will [generate a provenance statement](https://docs.npmjs.com/generating-provenance-statements) for your releases if the following conditions are met:
+
+ - In the `.yml` file that configures your release action:
+   - Add `provenance: true` to your list of **inputs**
+   - Add `id-token: write` to your list of **permissions**
+ - Ensure your CI runner uses NPM >= 9.5.0 (should be the default if Node >= 18)
+ - Ensure your `package.json` is complete and correct. NPM will cancel the release with specific errors if it finds a problem. Requirements may change in future NPM releases but include things like a `"repository"` field with `"url"` property matching the format `"git+https://github.com/user/repo"`.
+
+If `provenance: true` is in your YML inputs and any condition isn't met, the release will cancel with an error. 
+
+If you run a release without `provenance: true` when a previous release had provenance, the provenance banner on your package's NPM landing page will be removed but the banner will remain on the subpage for the previous release.
```

Comment:
this is certainly good to know, but is it really something that we need to document or shall we just redirect the user to the official docs? I mean we are not the owners of this information, and it may change

Reply:
👍 I've reworked this section in the ["Rework readme advice" commit](https://github.com/nearform-actions/optic-release-automation-action/pull/294/commits/11cc1d5499e5f719bee22d4e6120d25a7e65c933)